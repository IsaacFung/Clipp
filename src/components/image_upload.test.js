import React from "react";
import renderer from 'react-test-renderer';

import ImageUpload from './image_upload'

it('renders correctly', () => {
    const tree = renderer.create(<ImageUpload/>).toJSON();
    expect(tree).toMatchSnapshot();
});