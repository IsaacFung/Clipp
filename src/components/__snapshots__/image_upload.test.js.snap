// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={Object {}}
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "borderRadius": 15,
          "height": 100,
          "resizeMode": "stretch",
          "width": 100,
        }
      }
    />
  </View>
</View>
`;
